source/lex.tex

```diff
@@ -169,16 +169,16 @@
 All preprocessing directives are then deleted.
 
 \item
-For a sequence of two or more adjacent \grammarterm{string-literal} tokens,
+For a sequence of two or more adjacent \grammarterm{string-literal} preprocessing tokens,
 a common \grammarterm{encoding-prefix} is determined
 as specified in \ref{lex.string}.
-Each such \grammarterm{string-literal} token is then considered to have
+Each such \grammarterm{string-literal} preprocessing token is then considered to have
 that common \grammarterm{encoding-prefix}.
 
 \item
-Adjacent \grammarterm{string-literal} tokens are concatenated\iref{lex.string}.
+Adjacent \grammarterm{string-literal} preprocessing tokens are concatenated\iref{lex.string}.
 
-\item Whitespace characters separating tokens are no longer
+\item Whitespace characters separating preprocessing tokens are no longer
 significant. Each preprocessing token is converted into a
 token\iref{lex.token}. The resulting tokens
```

Comment:
Agreed with the rest of the changes, but this last paragraph would benefit from a bit of rephrasing:

"Each preprocessing token is converted into a token\iref..; whitespace characters separating tokens are no longer significant. The resulting tokens..."

Reply:
Rebased, and now this comment is already resolved in `main`, simplifying the remaining PR considerably.